packages/middleware-retry/src/defaultStrategy.ts

```diff
@@ -106,9 +118,9 @@ export class StandardRetryStrategy implements RetryStrategy {
     while (true) {
       try {
         if (HttpRequest.isInstance(request)) {
-          request.headers["amz-sdk-request"] = `attempt=${attempts + 1}; max=${
-            this.maxAttempts
-          }`;
+          request.headers["amz-sdk-request"] = `attempt=${
+            attempts + 1
+          }; max=${maxAttempts}`;
         }
```

Comment:
nit: Attempt {attempt} of {max}. I would file a suggestion, but github isn't allowing me to do a multiline one. I dont really feel strongly about this at all.

Reply:
The existing header is a standard followed across SDKs and services.
Details in internal issue JS-1816